From 79ff7cb539374f034aa45beb23a4dbda435e79b0 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Fri, 15 Sep 2023 22:54:06 +0200
Subject: [PATCH 11/15] Revert "sphinx: kfigure.py: Convert outdir to str
 before using len"

This reverts commit 37d1bd4999f06c7c4b3094cb74b7614c552ecc1d.

Should be unnecessary as of Sphinx 7.2.3.
---
 Documentation/sphinx/kfigure.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/sphinx/kfigure.py b/Documentation/sphinx/kfigure.py
index 13e885bbd..cefdbb7e7 100644
--- a/Documentation/sphinx/kfigure.py
+++ b/Documentation/sphinx/kfigure.py
@@ -309,7 +309,7 @@ def convert_image(img_node, translator, src_fname=None):
     if dst_fname:
         # the builder needs not to copy one more time, so pop it if exists.
         translator.builder.images.pop(img_node['uri'], None)
-        _name = dst_fname[len(str(translator.builder.outdir)) + 1:]
+        _name = dst_fname[len(translator.builder.outdir) + 1:]
 
         if isNewer(dst_fname, src_fname):
             kernellog.verbose(app,
-- 
2.42.0

